HolisticSolutionSDK/Appodeal/HSAppodealConnector.swift

```diff
@@ -38,3 +43,25 @@ extension HSAppodealConnector: HSAdvertising {
         Appodeal.setExtras(["keywords": keywords])
     }
 }
+
+extension HSAppodealConnector: HSAnalyticsService {
+    func trackEvent(_ event: String, customParameters: [String : Any]?) {
+        Appodeal.setExtras([event : customParameters ?? [:]])
+    }
+    
+    func trackInAppPurchase(_ purchase: HSPurchase) {
+        guard
+            trackingEnabled,
+            let value = NumberFormatter().number(from: purchase.price)
```

Comment:
This approach seems to be incorrect because the price is locale specific.
Please check [Apple's doc](https://developer.apple.com/documentation/storekit/skproduct/1506094-price)